static/app/components/events/eventCause.tsx

```diff
@@ -66,12 +59,12 @@ class EventCause extends Component<Props, State> {
     return (
       <DataSection>
         <CauseHeader>
-          <h3>
+          <h3 data-test-id="event-cause">
```

Comment:
do we need this `data-test-id`? since we are using RTL, we could use the queries `getByText` or `getByRole`

Reply:
I tried that and it seems that this component renders multiple "Suspect Commits" texts. The old test was asserting that EventCause renders (with enzyme), so this best reflects that